src/lib/components/ProjectsPage/ProjectsPage.consts.ts

```diff
@@ -67,3 +67,9 @@ export const projectsTableColumns = fieldsNamesToColumns(
   ],
   PROJECT_DISPLAY_TEXTS.he.fields,
 );
+
+export const searchBy = [
```

Comment:
can we make the name more ProjectsPage related ? (e.g. projectsTableSearchBy)

Reply:
sounds resonable ill change it